CHANGELOG.md

```diff
@@ -249,12 +249,13 @@ Current
         synchronized. Since threads need to acquire a write lock before
         doing anything else anyway, the method is still effectively 
         synchronized.
-        
-### Known Issues:
+- [Made now required constructor for ArithmeticMaker with rounding public](https://github.com/yahoo/fili/pull/148)
```

Comment:
Because the constructor was protected so it was actually unusable despite being mandatory?